cmd/bandd/cmd/root.go

```diff
@@ -84,27 +84,21 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	rootCmd.AddCommand(
 		InitCmd(band.NewDefaultGenesisState(), band.DefaultNodeHome),
-		genutilcli.CollectGenTxsCmd(banktypes.GenesisBalancesIterator{}, band.DefaultNodeHome),
-		genutilcli.GenTxCmd(
-			band.ModuleBasics,
-			encodingConfig.TxConfig,
-			banktypes.GenesisBalancesIterator{},
-			band.DefaultNodeHome,
+		genesisCommand(
+			encodingConfig,
+			AddGenesisDataSourceCmd(band.DefaultNodeHome),
+			AddGenesisOracleScriptCmd(band.DefaultNodeHome),
 		),
-		genutilcli.ValidateGenesisCmd(band.ModuleBasics),
-		AddGenesisAccountCmd(band.DefaultNodeHome),
-		AddGenesisDataSourceCmd(band.DefaultNodeHome),
-		AddGenesisOracleScriptCmd(band.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
 		// testnetCmd(band.ModuleBasics, banktypes.GenesisBalancesIterator{}),
 		debug.Cmd(),
 		config.Cmd(),
+		pruning.Cmd(newApp, band.DefaultNodeHome),
+		// support oracle file folder
```

Comment:
is this TODO or already supported?

Reply:
ok

Reply:
I think it is the comment that is not intentionally put there